### PR TITLE
Route and container login

### PR DESCRIPTION
**Pull request by marco-rozo**
# Description

# Fixes 
 - https://trello.com/c/EkpjbLPh/35-front-integra%C3%A7%C3%A3o-login
 - https://trello.com/c/vPNZMz4c/34-front-cria%C3%A7%C3%A3o-do-singleton-do-axios
 - https://trello.com/c/70D7ZrVm/33-front-defini%C3%A7%C3%A3o-do-env
 - https://trello.com/c/0Prjf1Dy/32-front-refinamento-login-page
 - https://trello.com/c/z5OOItIe/23-front-l%C3%B3gica-autentica%C3%A7%C3%A3o
 - https://trello.com/c/DNb8l5Cl/10-front-cria%C3%A7%C3%A3o-das-rotas
 - https://trello.com/c/wTVQ9FJ3/28-front-login-page

# Screenshot 
 - [Login Page] (https://i.imgur.com/nFY9dez.png)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,12 +11,14 @@
     "lint": "lint-staged"
   },
   "dependencies": {
+    "phosphor-react": "^1.4.1",
```
Comment (Minozzzi) [inline package.json]: A gente ta usando icons do material ui para manter padronizado com os components, faz sentido usar essa?

File: src/main.tsx [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+import { Router } from './routes'
 import { ResetStyle, theme } from '@/styles'
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <ThemeProvider theme={theme}>
     <React.StrictMode>
-      <App />
-      <ResetStyle />
+      <BrowserRouter>
+        {/* <AuthProvider> */}
```
Comment (Minozzzi) [inline src/main.tsx]: Remover comentários

**Review by Minozzzi — CHANGES_REQUESTED** [review]

**Comment by marco-rozo** [comment]
ok

**Review by Minozzzi — APPROVED** [review]

**Review by viniciuspegorini — APPROVED** [review]
ok!

**Pull request merged**